### ISSUE TITLE
Error loading segment anything model running on CPU

### ISSUE THREAD

**Issue by ptlzon** [SchmollerLab/Cell_ACDC#480]
### Description:

After setting segment_anything parameters and clicking OK, this error appeared. I install the necessary libraries successfully. For torchvision, I only installed CPU version.

![image](https://github.com/SchmollerLab/Cell_ACDC/assets/157567305/54d96165-33d0-40ad-ada2-b35e65a39241)

### Terminal output:

```Shell
19-02-2024, 16:42:35 - cellacdc-logger-acdc_gui - INFO:
------------------------
Initializing GUI v1.4.20
------------------------

19-02-2024, 16:42:35 - cellacdc-logger-acdc_gui - INFO:
------------------------
Initiliazing profilers...
------------------------

19-02-2024, 16:42:35 - cellacdc-logger-acdc_gui - INFO:
------------------------
GUI ready.
------------------------

19-02-2024, 16:42:38 - cellacdc-logger-acdc_gui - INFO:
------------------------
Asking to select a folder path...
------------------------

19-02-2024, 16:42:40 - cellacdc-logger-acdc_gui - INFO:
------------------------
Loading from D:/tmp/grayscale/train/20240219_161436_acdc/Images
------------------------

19-02-2024, 16:42:40 - cellacdc-logger-acdc_gui - INFO:
------------------------
Using .tif file (D:/tmp/grayscale/train/20240219_161436_acdc/Images\22-696-2_001.tif)...
------------------------

19-02-2024, 16:42:40 - cellacdc-logger-acdc_gui - INFO:
------------------------
Reading  channel metadata...
------------------------

19-02-2024, 16:42:40 - cellacdc-logger-acdc_gui - INFO:
------------------------
Lazy loader paused.
------------------------

19-02-2024, 16:42:40 - cellacdc-logger-acdc_gui - INFO:
------------------------
Closing lazy loader...
------------------------

19-02-2024, 16:42:40 - cellacdc-logger-acdc_gui - INFO:
------------------------
Segmentation filename: D:/tmp/grayscale/train/20240219_161436_acdc/Images\22-696-2_001_segm.npz
------------------------

19-02-2024, 16:42:46 - cellacdc-logger-acdc_gui - INFO:
------------------------
Loading 20240219_161436_acdc\Images\22-696-2_001.tif...
------------------------

19-02-2024, 16:42:46 - cellacdc-logger-acdc_gui - INFO:
------------------------
Loaded paths:
Segmentation file name: 22-696-2_001_segm.npz
ACDC output file name 22-696-2_001_acdc_output.csv
------------------------

19-02-2024, 16:42:46 - cellacdc-logger-acdc_gui - INFO:
------------------------
Full dataset shape = Not Loaded
------------------------

19-02-2024, 16:42:46 - cellacdc-logger-acdc_gui - INFO:
------------------------
Loaded dataset shape = (1, 2304, 3072)
------------------------

19-02-2024, 16:42:46 - cellacdc-logger-acdc_gui - INFO:
------------------------
Number of frames = 1
------------------------

19-02-2024, 16:42:46 - cellacdc-logger-acdc_gui - INFO:
------------------------
Number of z-slices per frame = 1
------------------------

19-02-2024, 16:42:46 - cellacdc-logger-acdc_gui - INFO:
------------------------
Creating graphical items...
------------------------

19-02-2024, 16:42:47 - cellacdc-logger-acdc_gui - INFO:
------------------------
Initializing measurements...
------------------------

19-02-2024, 16:42:47 - cellacdc-logger-acdc_gui - INFO:
------------------------
Setting GUI mode to "Snapshots"...
------------------------

19-02-2024, 16:42:47 - cellacdc-logger-acdc_gui - INFO:
------------------------
20240219_161436_acdc || Basename: 22-696-2_001 || Segmented channel:  || Segmentation file name: _segm.npz
------------------------

19-02-2024, 16:42:47 - cellacdc-logger-acdc_gui - INFO:
------------------------
Autosaving worker started.
------------------------

19-02-2024, 16:42:54 - cellacdc-logger-acdc_gui - INFO:
------------------------
Importing segment_anything...
------------------------

19-02-2024, 16:43:04 - cellacdc-logger-acdc_gui - ERROR:
------------------------
Traceback (most recent call last):
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\core.py", line 1636, in segm_model_segment
    lab = model.segment(image, **model_kwargs)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\models\segment_anything\acdcSegment.py", line 68, in segment
    labels = self._segment_2D_image(image)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\models\segment_anything\acdcSegment.py", line 74, in _segment_2D_image
    img = myutils.to_uint8(image)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\myutils.py", line 1733, in to_uint8
    img = np.round(img_to_float(img)*255).astype(np.uint8)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\myutils.py", line 1771, in img_to_float
    if img_max <= 1.0 and issubclass(input_img_dtype, (np.floating, float)):
TypeError: issubclass() arg 1 must be a class

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\__init__.py", line 366, in inner_function
    result = func(self, *args)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\gui.py", line 14008, in workerCritical
    raise error
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\workers.py", line 45, in run
    func(self)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\workers.py", line 766, in run
    _lab = core.segm_model_segment(
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\core.py", line 1638, in segm_model_segment
    lab = model.segment(image, frame_i, **model_kwargs)
TypeError: segment() got multiple values for argument 'automatic_removal_of_background'

------------------------
Traceback (most recent call last):
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\core.py", line 1636, in segm_model_segment
    lab = model.segment(image, **model_kwargs)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\models\segment_anything\acdcSegment.py", line 68, in segment
    labels = self._segment_2D_image(image)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\models\segment_anything\acdcSegment.py", line 74, in _segment_2D_image
    img = myutils.to_uint8(image)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\myutils.py", line 1733, in to_uint8
    img = np.round(img_to_float(img)*255).astype(np.uint8)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\myutils.py", line 1771, in img_to_float
    if img_max <= 1.0 and issubclass(input_img_dtype, (np.floating, float)):
TypeError: issubclass() arg 1 must be a class

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\__init__.py", line 366, in inner_function
    result = func(self, *args)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\gui.py", line 14008, in workerCritical
    raise error
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\workers.py", line 45, in run
    func(self)
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\workers.py", line 766, in run
    _lab = core.segm_model_segment(
  File "D:\miniconda3_windows\envs\acdc\lib\site-packages\cellacdc\core.py", line 1638, in segm_model_segment
    lab = model.segment(image, frame_i, **model_kwargs)
TypeError: segment() got multiple values for argument 'automatic_removal_of_background'
19-02-2024, 16:43:04 - cellacdc-logger-acdc_gui - INFO:
------------------------
====================================
WARNING: Cell-ACDC is in error state. Please, restart.
====================================
------------------------
```

### Version information:

```Shell
(acdc) D:\tmp>python -c "import sys; print(sys.version); import platform; print(platform.platform()); import cellacdc; print(f'Cell-ACDC version: {cellacdc.__version__}')"
3.9.18 | packaged by conda-forge | (main, Dec 23 2023, 16:29:04) [MSC v.1929 64 bit (AMD64)]
Windows-10-10.0.19045-SP0
Cell-ACDC version: 1.4.20
```

**Comment by ElpadoCan** [comment]
Hi @ptlzon, as for the other issue, this should also be fixed in the newest release version `v1.4.21`. 

To upgrade, activate the `acdc` environment and run the command `python -m pip install --upgrade cellacdc`. Let me know if it works, thanks! 

Best,
Francesco

**Comment by ptlzon** [comment]
Hi @ElpadoCan 

Thanks for the quick fix. For SAM, the fix did it. The model could be loaded correctly now.
Unfortunately I cannot add more than one point using clicks (I want to add click points and get segmentation results from that multi points).
I will create another issue.

Best,
Lam
